eth/tracers/api.go

```diff
@@ -200,6 +200,7 @@ type StdTraceConfig struct {
 
 // txTraceResult is the result of a single transaction trace.
 type txTraceResult struct {
+    TxHash common.Hash `json:"txHash,omitempty"` // transaction hash
```

Comment:
Does `omitEmpty` work as intended on the empty-hash?

Reply:
Thank you for the review. I realized that common.Hash is of type [32]byte, so it cannot have an empty value in a byte array. I will modify that.

Reply:
fixed it!